manifests/integreatly-3scale/0.10.0-mas/apps.3scale.net_apimanagerbackups.yaml

```diff
@@ -0,0 +1,99 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Do we need to maintain the bundles in the repo still? 🤔

Reply:
I think these are needed by make release/prepare for the available images for scanning. It's a little hard to see exactly what was made by the pipeline as the old manifests pipeline made this PR and the initial commits up to the last one. I think we should disable the old pipeline

Reply:
hey, yes we do. Happy to explain offline